spec/models/search_builder_spec.rb

```diff
@@ -28,5 +28,13 @@
         expect(solr_parameters[:mm]).to be_nil
       end
     end
+
+    context "when the query isn't a string" do
+      let(:query) { { q: { a: 1 } } }
```

Comment:
How does this query occur in the real world?

Reply:
When an item is embedded on a page in spotlight.

Reply:
Is there a page I can go to in order to see this?

Reply:
Merging this, but we believe this may just be cover for a deeper issue https://github.com/sul-dlss/dlme/issues/1889